tests/test_envs.py

```diff
@@ -0,0 +1,33 @@
+import gym
+import pytest
+
+from imitation.envs import ENV_NAMES
+from imitation.util import make_vec_env
+
+PARALLEL = [False, True]
+
+try:
```

Comment:
codecov will complain as one side of this branch isn't taken so we don't achieve 100% test coverage. Add a "# pragma: nocover" comment.

Reply:
`pytype` is also unhappy about this and will need a comment.

In general I'm uneasy about disabling some tests (e.g. I'm pretty sure this test will *not* pass right now, but it's still green on Travis because it skips the test). However, getting MuJoCo working on Travis is a bit of a pain, and some people may not have license keys so probably best to avoid it being a forced dependency.

Reply:
stable-baselines zoo [uses `pybullet`](https://github.com/araffin/rl-baselines-zoo#pybullet-environments) for its "mujoco" environments. It would be neat if this project usurped Mujoco

Reply:
I think I'll leave the `skipif` test in for now. Arguably we could also omit the test because we shouldn't ever make changes to the AIRL environments